spinalcordtoolbox/download.py

```diff
@@ -39,8 +39,7 @@
     },
     "PAM50": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20201104/PAM50-r20201104.zip",
-            "https://osf.io/download/5fa21326a5bb9d00610a5a21/",
+            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20220527/PAM50-r20220527.zip",
```

Comment:
Note: Currently, this `r20220527` release is not up-to-date -- it contains the older, "uncorrected" histology files. So, we will need to create a new release that includes the corrected histology files.

However, we also need to coordinate this PAM50 release with the [Phillips Lab level files update](https://github.com/spinalcordtoolbox/PAM50/pull/3), too.  (Ideally, we would create a release that contains both updates.)